src/imitation/scripts/train_rl.py

```diff
@@ -80,6 +82,10 @@ def train_rl(
 
     Returns:
         The return value of `rollout_stats()` using the final policy.
+
+    Raises:
```

Comment:
In general, on the one hand, I think it's dubious to use VecNormalize. On the other hand, I think we should track the reward fed into the RL algorithm.

For not using `VecNormalize`:
- Let's say the transfer distribution for learned reward shifted a lot from a scale of mean 0 to 100. It might be better to shift the mean back to 0 rather than only rescaling its variance using VecNormalize. We might also want to do it for train_rl using ground truth reward, otherwise the reward scale will be naturally different between GT reward and learned reward (normalized from train norm stats).
- For a reward with arbitrary scale of mean, we are not sure how to set the `clip_reward` scale.

My current best guess are to
1. Keep the norm stats updated during retraining/transfer.
2. Or wrap the loaded reward model with ... another `RewardWrapper` where we can easily choose which norm we wanna use between RunningNorm and EWMA.

At the same time, I think it's okay to keep the VecNormalize option on until we decide whether to replace VecNormalize.

Reply:
Yawen and I had a discussion about having this be an error or a warning https://github.com/HumanCompatibleAI/imitation/pull/440#discussion_r885062805. I can change it back to being a warning fairly easily.

Reply:
Based on our last conversations, we might want to convert it back to a warning for the purpose of this repo.

Reply:
I converted it back to a warning.